buildenv/jenkins/jobs/pipelines/Pipeline-Build-Test-All

```diff
@@ -56,22 +57,24 @@ timeout(time: 10, unit: 'HOURS') {
         // update build description
         currentBuild.description = "OpenJ9: ${OPENJ9_SHA}<br/>OMR: ${OMR_SHA}"
 
-        SDK_VERSIONS.each { SDK_VERSION ->
-            def OPENJDK_REPO = OPENJDK_REPOS["${SDK_VERSION}"]
-            def OPENJDK_BRANCH = OPENJDK_BRANCHES["${SDK_VERSION}"]
-            def OPENJDK_SHA = buildFile.get_sha(OPENJDK_REPO, OPENJDK_BRANCH)
+        echo "Building OpenJ9 extensions for OpenJDK"
```

Comment:
I don't think this line is necessary or accurate.

Reply:
removed